pages/api/graphql-rest.page.ts

```diff
@@ -99,6 +132,35 @@ class MpdxRestApi extends RESTDataSource {
     }
   }
 
+  async createExportedContacts(
+    mailing: boolean,
+    format: string,
+    filter: {
+      account_list_id: string;
+      newsletter: string;
+      status: string;
+    },
+    labelType?: string | null,
+    sort?: string | null,
+  ) {
+    let pathAddition = '';
+    if (mailing) {
+      pathAddition = '/mailing';
+    }
```

Comment:
```suggestion
    const pathAddition = mailing ? '/mailing' : '';
```
I prefer the immutable syntax. Trivial though.